eox_nelp/settings/common.py

```diff
@@ -60,9 +60,9 @@ def plugin_settings(settings):
     if find_spec('eox_audit_model') and EOX_AUDIT_MODEL_APP not in settings.INSTALLED_APPS:
         settings.INSTALLED_APPS.append(EOX_AUDIT_MODEL_APP)
 
-
     try:
-        payments_notifications_context = 'eox_nelp.payment_notifications.context_processor.payments_notifications_context'
+        payments_notifications_context = 'eox_nelp.payment_notifications.context_processor.payments_notifications_context'  # pylint: disable=line-too-long
```

Comment:
Does this really need a  skip of `line-too-long` or could be split into more lines?